ext/mysqli/tests/mysqli_stmt_bind_result_format.phpt

```diff
@@ -227,7 +227,7 @@ memory_limit=83886080
                 $current_targets = mt_rand(-100000, 100000) / 10;
                 do {
                     $trend = (mt_rand(0, 3) > 1) ? (mt_rand(-10000, 10000) / 100) : 'NULL';
-                } while (isset($values[$trend]));
+                } while (isset($values[(int)$trend]));
```

Comment:
I think in this case `(string)$trend` might have been intended, as `$trend` is used as a float value here.